core/src/err/mod.rs

```diff
@@ -779,6 +779,142 @@ pub enum Error {
 	/// The db is running without an available storage engine
 	#[error("The db is running without an available storage engine")]
 	MissingStorageEngine,
+
+	/// The requested analyzer already exists
+	#[error("The analyzer '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	AzAlreadyExists {
+		value: String,
+	},
+
+	/// The requested database already exists
+	#[error("The database '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	DbAlreadyExists {
+		value: String,
+	},
+
+	/// The requested event already exists
+	#[error("The event '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	EvAlreadyExists {
+		value: String,
+	},
+
+	/// The requested field already exists
+	#[error("The field '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	FdAlreadyExists {
+		value: String,
+	},
+
+	/// The requested function already exists
+	#[error("The function 'fn::{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	FcAlreadyExists {
```

Comment:
`FunctionAlreadyExists`? If you really need to abbreviate this, `FnAlreadyExists` will resonate better with Rust users.